dev/NavigationView/TestUI/NavigationViewRS4Page.xaml

```diff
@@ -237,7 +237,6 @@
                                     <VisualState x:Name="TitleBarVisible" />
                                     <VisualState x:Name="TitleBarCollapsed">
                                         <VisualState.Setters>
-                                            <Setter Target="PaneButtonGrid.Margin" Value="0,32,0,0"/>
```

Comment:
Backout this change, same reason as NavigationViewRS3Page.xaml

Reply:
Thanks, done ;) !